pipelines/broad/internal/rna_seq/BroadInternalRNAWithUMIs.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.20
+2022-12-07 (Date of Last Commit)
+
+* In MergeMetrics task, convert "?" to "NaN" and round some median metrics to integers.
```

Comment:
```suggestion
* In MergeMetrics task, convert \"?\" to \"NaN\" and round some median metrics to integers.
```

Reply:
We have to escape double quotes in our changelogs or else the jenkins release fails (i think it has a hard time parsing the quotes)